pkg/cstor/pv_operation.go

```diff
@@ -132,8 +135,6 @@ func (p *Plugin) getVolumeForRemoteRestore(volumeID, snapName string) (*Volume,
 		return nil, err
 	}
 
-	p.volumes[vol.volname] = vol
```

Comment:
why did we removed this?

Reply:
`createPVC` take care of updating p.volumes with new volume. But better to keep this here to make it similar to `getVolumeForLocalRestore`. cc: @sonasingh46 (to take this in next PR).